doc/source/serve/advanced.rst

```diff
@@ -0,0 +1,132 @@
+=====================================
+Advanced Topics, Configurations & FAQ
+=====================================
+
+Ray Serve has a number of knobs and tools for you to tune for your particular workload. 
+All Ray Serve advanced options and topics are covered on this page aside from the 
+fundamentals of :doc:`deployment`. For a more hands on take, please check out the :ref:`serve-tutorials`.
+
+There are a number of things you'll likely want to do with your serving application including
+scaling out, splitting traffic, or batching input for better response performance. To do all of this,
+you will create a ``BackendConfig``, a configuration object that you'll use to set 
+the properties of a particular backend.
+
+.. contents::
+
+Scaling Out
+===========
+
+To scale out a backend to multiple workers, simplify configure the number of replicas.
+
+.. code-block:: python
+
+  config = {"num_replicas": 2}
+  serve.create_backend("my_scaled_endpoint_backend", handle_request, config=config)
+
+This will scale out the number of workers that can accept requests.
+
+Using Resources (CPUs, GPUs)
+============================
+
+To assign hardware resource per worker, you can pass resource requirements to
+``ray_actor_options``. To learn about options to pass in, take a look at
+:ref:`Resources with Actor<actor-resource-guide>` guide.
+
+For example, to create a backend where each replica uses a single GPU, you can do the
+following:
+
+.. code-block:: python
+
+  options = {"num_gpus": 1}
```

Comment:
Should we rename the keyword argument to be `config` as opposed to `options` to maintain this consistency as well?

```
serve.set_backend_config("my_scaled_endpoint_backend", handle_request, config=config)
serve.create_backend("my_gpu_backend", handle_request, ray_actor_options=config)
```

change to:

```
serve.set_backend_config("my_scaled_endpoint_backend", handle_request, config=config)
serve.create_backend("my_gpu_backend", handle_request, ray_actor_config=config)
```

Reply:
I have no idea if this is a good idea or not, just a thought.

Reply:
Possibly. The reason it is called options instead of config config right now is because in ray you pass them in as flags to `Actor.options(...)` and there is no notion of an "actor config." I don't feel strongly either way.

Reply:
ok, let's keep it as is then!